middleware/index.js

```diff
@@ -1,41 +1,17 @@
-const moment = require('moment');
-
 const Models = require('../models');
 const Performance = new Models.Performance();
 const { logger } = require('../utils');
 
-function currentPerformanceWithinTimeFrame(performance) {
-  const now = new Date().toJSON();
-
-  return moment(performance.closeAt).add({ days: 1 }).isBefore(moment(now));
-}
-
-const currentPerformance = (req, res, next) => {
-  Performance.findCurrent()
-  .then((performance) => {
-    const now = moment(), { openat, closeat } = performance;
-
-    if (moment(openat).isBefore(now) && now.isBefore(moment(closeat))) {
-      req.session.currentPerformance = performance;
-    }
-    next();
-  })
-  .catch((err) => {
-    logger.errorLog('Middleware.currentPerformance', err);
-    next();
-  });
-};
-
 /* eslint-disable consistent-return */
```

Comment:
I'm not recently familiar with how Express middleware works. When you `return next()`, does it use the return value from `next` in whatever called the middleware? Why do you sometimes ignore the return value and sometimes `return` it?